toqito/channel_props/is_quantum_channel.py

```diff
@@ -25,11 +25,11 @@ def is_quantum_channel(
     2. :math:`\Phi` is trace preserving.
 
     Examples
-    ==========
+    ========
     We can specify the input as a list of Kraus operators. Consider the map :math:`\Phi` defined as
 
     .. math::
-        \Phi(X) = X - U X U^*
+        \Phi(X) = X + U X U^*
```

Comment:
I don't know if this is supposed to be a typo. `Phi(X)` evaluates to a matrix full of zeroes. There were some errors raised when I used `is_quantum_channel` in this case. So, I changed the `-` to a `+`. 

`IndexError: Subsystem index 1 out of range for 1 systems total.`

Reply:
Just to check here, what is the example you're running that is causing this outcome? Trying to replicate on my end.

Reply:
If I use `phi = x - np.matmul(u, np.matmul(x, np.conjugate(u)))` in the example of L43-51, I get `IndexError: Subsystem index 1 out of range for 1 systems total.`

This is because `phi = x - np.matmul(u, np.matmul(x, np.conjugate(u)))` is a zero matrix. 

```
[[-2.23711432e-17  2.22044605e-16]
 [ 2.22044605e-16 -2.23711432e-17]]
```
Whereas  `phi = x + np.matmul(u, np.matmul(x, np.conjugate(u)))` is Pauli X without the normalization factor.
```
[[2.23711432e-17 2.00000000e+00]
 [2.00000000e+00 2.23711432e-17]]
```

Reply:
Hmm, I think the issue is a bit deeper than that. It's actually in the `is_trace_preserving` function. 

Indeed, in `is_trace_preserving.py` if you alter `from picos import partial_trace` to `from toqito.channels import partial_trace` this eliminates the first index error you were seeing.

Reply:
Thanks for catching this! 

I thought this error could be due to some function inverting the matrix. For a matrix full of zeroes, this would have created more issues.